core-web/libs/sdk/client/src/lib/client/sdk-js-client.ts

```diff
@@ -19,17 +19,15 @@ export interface ClientConfig {
     /**
      * The id of the site you want to interact with.
      *
-     * @description to get the site id, go to the site you want to interact with and copy the id from the History tab
-     *
+     * @description To get the site id, go to the site you want to interact with and copy the id from the History tab.
      * @type {string}
-     * @required
+     * @optional
      */
     siteId?: string;
     /**
      * The authentication token to use for the requests. If not provided, it will fallback to default site.
```

Comment:
"If not provided, it will fallback to default site."... what?

Reply:
Wonder who added that, hahaha! Let me fix it